docs/experiments-atomic-bm25.md

```diff
@@ -0,0 +1,71 @@
+
+# Pyserini: Reproducing AToMiC BM 25 Baselines
+
+Pyserini provides the following pre-built indexes for the AToMiC dataset to reproduce the baselines in the [AToMiC paper](https://arxiv.org/pdf/2304.01961.pdf):
+- `lucene-index.atomic.text.flat.small.validation`
+- `lucene-index.atomic.text.flat.base`
+- `lucene-index.atomic.text.flat.large`
+- `lucene-index.atomic.image.flat.small.validation`
+- `lucene-index.atomic.image.flat.base`
+- `lucene-index.atomic.image.flat.large`
+
+## Data Prep
+We need the topic files (`validation.text.search.jsonl` and `validation.image-caption.search.jsonl`) and the qrels files (`validation.qrels.t2i.projected.trec` and `validation.qrels.i2t.projected.trec`) to reproduce the baselines given in the paper.
+
+If you have a dev installation of `pyserini`, the required files are located under `pyserini/tools/topics-and-qrels/`. Otherwise, you can download them at [this repository](https://huggingface.co/spaces/dlrudwo1269/AToMiC_bm25_files/tree/main/). 
```

Comment:
Maybe add a few lines for explicit data prep, s.t. we can copy/paste the commands easily and check the results.

For example:
```bash
export DATA_DIR="https://huggingface.co/spaces/dlrudwo1269/AToMiC_bm25_files/resolve/main"

mkdir topics
wget ${DATA_DIR}/topics/validation.text.search.jsonl -P topics
wget ...

mkdir qrels
...
```